vagrant/README.md

```diff
@@ -21,7 +21,7 @@
 First - install: [Vagrant](https://www.vagrantup.com/docs/installation/), [VirtualBox](https://www.virtualbox.org/wiki/Downloads) and [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). Then you just need to execute the following commands (they should work on any flavour of Linux):
 
 To run Vagrant development environment, the `$SWAN_DEVELOPMENT_ENVIRONMENT` variable must be set.
-
+If you want to use your local Glide repository, the `$SHARE_GLIDE_CACHE` variable must be set.
```

Comment:
I would rephrase it to:
If you want to use you local Glide cache in the Vagrant development environment the `$SHARE_GLIDE_CACHE` variable must be set.